config/secrets.yml

```diff
@@ -3,6 +3,7 @@ default: &default
     cognito_aws_access_key_id: <%= ENV["COGNITO_AWS_ACCESS_KEY_ID"] %>
     cognito_aws_secret_access_key: <%= ENV["COGNITO_AWS_SECRET_ACCESS_KEY"] %>
     cognito_client_id: <%= ENV["AWS_COGNITO_CLIENT_ID"] %>
+    cognito_userpool_id: <%= ENV["COGNITO_USERPOOL_ID"] %>
```

Comment:
do we need the userpool_id for getting the user ID?

Reply:
No... Sorry this was added while I was investigating getting usernames in the event we need to see more than a GUID... If you do need to see more than a GUID then we'll need this for sure as it allows querying Cognito for user and group information.

Can remove for now but we will need it soon.